src/cljs/athens/devcards/athena.cljs

```diff
@@ -47,21 +50,125 @@
+
+
+
+(def container-style
+  {:width         "784px"
+   :border-radius "4px"
+   :box-shadow    [[(:64 DEPTH-SHADOWS) ", 0 0 0 1px " (opacify (:body-text-color HSL-COLORS) (first OPACITIES))]]
+   :display       "flex"
+   :flex-direction "column"
+   :background    (:app-bg-color HSL-COLORS)
+   :position      "fixed"
+   :overflow      "hidden"
+   :max-height    "60vh"
+   :top           "50%"
+   :left          "50%"
+   :transform     "translate(-50%, -50%)"
+   :z-index       2})
+
+
+(def athena-input-style
+  {:width "100%"
+   :border 0
+   :font-size      "38px"
+   :font-weight    "300"
+   :line-height    "49px"
+   :letter-spacing "-0.03em"
+   :border-radius "4px 4px 0 0"
+   :color          "#433F38"
+   :caret-color    (:link-color COLORS)
+   :padding "24px"
+   :cursor "text"
+   ::stylefy/mode {:focus {:outline "none"}
+                   "::placeholder" {:opacity (nth OPACITIES 2)}}})
+
+
+(def results-list-style
+  {:background    (:app-bg-color HSL-COLORS)
+   :overflow-y "auto"
+   :max-height "100%"})
+
+
+(def results-heading-style
+  {:padding "4px 18px"
+   :background (:app-bg-color HSL-COLORS)
+   :display "flex"
+   :position "sticky"
+   :top "0"
+   :justify-content "space-between"
+   :box-shadow [["0 1px 0 0 " (opacify (:body-text-color HSL-COLORS) 0.12)]]
+   :border-top [["1px solid" (opacify (:body-text-color HSL-COLORS) 0.12)]]})
+
+
+(def result-style
+  {:display "grid"
+   :grid-template "\"title icon\" \"preview icon\""
+   :grid-gap "0 12px"
+   :grid-template-columns "1fr auto"
+   :padding "8px 32px"
+   :background (opacify (:body-text-color HSL-COLORS) 0.02)
+   :transition "all .05s ease"
+   :border-top [["1px solid " (opacify (:body-text-color HSL-COLORS) 0.12)]]
+   ::stylefy/sub-styles {:title {:grid-area "title"
+                                 :font-size "16px"
+                                 :margin "0"
+                                 :color (:header-text-color COLORS)
+                                 :font-weight "500"}
+                         :preview {:grid-area "preview"
+                                   :white-space "wrap"
+                                   :word-break "break-word"
+                                   :overflow "hidden"
+                                   :text-overflow "ellipsis"
+                                   :display "-webkit-box"
+                                   :-webkit-line-clamp "1"
+                                   :-webkit-box-orient "vertical"
+                                   :color (opacify (:body-text-color COLORS) (nth OPACITIES 3))}
+                         :link-leader {:grid-area "icon"
+                                       :color "transparent"
+                                       :margin "auto auto"}}
+   ::stylefy/mode {:hover {:background (:link-color HSL-COLORS)
+                           :color (:app-bg-color COLORS)}}
+   ::stylefy/manual [[:&:hover [:.title :.preview :.link-leader {:color "inherit !important"}]]]})
```

Comment:
Why do we need to use `!important`?

Reply:
@tangjeff0 We don't, that shouldn't have stayed in there. I'll make a cleanup PR for this little stuff.